pkgs/servers/invidious/update.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p common-updater-scripts curl crystal crystal2nix jq git gnused nix nix-prefetch-git nix-update pkg-config
+git_url='https://github.com/iv-org/invidious.git'
+git_branch='master'
+git_dir='/var/tmp/invidious.git'
```

Comment:
```suggestion
tmp=$(mktemp -d)
trap 'rm -rf "$tmp"' exit
git_dir="$tmp/invidious.git"
```

This way there's a clean new temporary directory every time. Or is it intentional that you're reusing this temporary directory?

Reply:
I think [/var/tmp](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s15.html) is made for this.

Reply:
Interesting, I initially thought this was like how `/var/run` is symlinked to `/run` on NixOS, the same doesn't hold for `/var/tmp` and `/tmp` though. I'm thinking that `/var/tmp` is about the system-wide equivalent to the user-local `~/.cache`.

I think I'd probably still prefer the `XDG_CACHE_HOME` with the `~/.cache` fallback since it's configurable and a fairly popular [specification](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html), but I think `/var/tmp` is fine too.